compiled-lang/en.json

```
@@ -42,6 +42,8 @@
   "noRecommendationsDesc": "No known recommendations affect this cluster.",
   "noRecsError": "No recommendations available.",
```

Please, replace it to
```suggestion
  "noRecsError": "No recommendations available",
```